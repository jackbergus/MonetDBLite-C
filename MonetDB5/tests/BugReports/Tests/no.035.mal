# Monet Mil to Mal compiler
# Copyright (c) 2001-2004, CWI. All rights reserved.

#Predefined code segment
#The compiler can not guarantee an accurate compilation,
#because MIL unlike MAL is a dynamically typed language.
#A few guidelines to maximize usefullness.
#- make the type of variables explicit, in particular 'nil'
#- upon advice of M2m to remap identifiers, change it in your program directly
#use the "mil_schema" for additional support routines
#	io.print("# >Number:         35\n");
#	io.print("# >Category:       Monet_SGI\n");
#	io.print("# >Synopsis:       BATloop semantics\n");
#	io.print("# >Confidential:   no\n");
#	io.print("# >Severity:       serious\n");
#	io.print("# >Priority:       high\n");
#	io.print("# >Responsible:    mk\n");
#	io.print("# >State:          feedback\n");
#	io.print("# >Class:          sw-bug\n");
#	io.print("# >Submitter-Id:   unknown\n");
#	io.print("# >Arrival-Date:   Tue Sep 30 20:45:03 1997\n");
#	io.print("# >Originator:     mk\n");
#	io.print("# >Organization:\n");
#	io.print("# >Release:        970101\n");
#	io.print("# >Environment:\n");
#	io.print("# SGI\n");
#	io.print("# >Description:\n");
#	io.print("# Since yesterday the semantics has changed\?\n");
#	io.print("# > b:= view_bbp_name;\n");
#	io.print("# > b@batloop(){ printf(\"%s\\n\",$t);}\n");
#	io.print("# !ERROR: interpret: no matching iterator to 'batloop(bat[int,str], ptr)'.\n");
#	io.print("var b:= view_bbp_name();");
	b := bbp.getNames());
#	io.print("b@batloop(){ printf(\"%s\\n\",$t);}");
barrier	(aa_a,aa_aH,aa_aT) := newIterator(b);
	ba_a := printf("%s\n",aa_aT);
redo	(aa_a,aa_aH,aa_aT) := hasMoreElements(b);
exit	(aa_a,aa_aH,aa_aT) ;
#	io.print("quit();");
	ca_a := clients.quit();
#Identifer 'view_bbp_name' mapped to 'bbp.getNames()'
