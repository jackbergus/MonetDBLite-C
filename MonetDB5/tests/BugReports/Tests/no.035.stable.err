stderr of test 'no.035` in directory 'monetdb5/tests/BugReports` itself:


# 13:14:58 >  
# 13:14:58 >  Mtimeout -timeout 60 Mserver "--config=/ufs/sjoerd/src/MonetDB/monet-merge/build/conf/monet.conf" --debug=10 --set "monet_mod_path=/ufs/sjoerd/src/MonetDB/monet-merge/build/src/modules/plain/.libs:/ufs/sjoerd/src/MonetDB/monet-merge/build/src/modules/contrib/.libs:/ufs/sjoerd/src/MonetDB/monet-merge/build/src/modules/calibrator/.libs:/ufs/sjoerd/src/MonetDB/monet-merge/build/src/mapi/.libs" --set "gdk_dbfarm=/ufs/sjoerd/src/MonetDB/monet-merge/build/dbfarm" --set "sql_logdir=/ufs/sjoerd/src/MonetDB/monet-merge/build/log" --set mapi_port=59517 --set sql_port=56441 --set monet_prompt= --trace --dbname=mTests_tests_BugReports  < no.035.milS
# 13:14:58 >  


# 13:14:58 >  
# 13:14:58 >  Done.
# 13:14:58 >  

