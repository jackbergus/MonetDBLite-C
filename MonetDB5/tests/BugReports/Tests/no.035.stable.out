stdout of test 'no.035` in directory 'monetdb5/tests/BugReports` itself:


# 13:14:58 >  
# 13:14:58 >  Mtimeout -timeout 60 Mserver "--config=/ufs/sjoerd/src/MonetDB/monet-merge/build/conf/monet.conf" --debug=10 --set "monet_mod_path=/ufs/sjoerd/src/MonetDB/monet-merge/build/src/modules/plain/.libs:/ufs/sjoerd/src/MonetDB/monet-merge/build/src/modules/contrib/.libs:/ufs/sjoerd/src/MonetDB/monet-merge/build/src/modules/calibrator/.libs:/ufs/sjoerd/src/MonetDB/monet-merge/build/src/mapi/.libs" --set "gdk_dbfarm=/ufs/sjoerd/src/MonetDB/monet-merge/build/dbfarm" --set "sql_logdir=/ufs/sjoerd/src/MonetDB/monet-merge/build/log" --set mapi_port=59517 --set sql_port=56441 --set monet_prompt= --trace --dbname=mTests_tests_BugReports  < no.035.milS
# 13:14:58 >  

# Monet Database Server V4.3.11
# Copyright (c) 1993-2004, CWI. All rights reserved.
# compiled for i686-pc-linux-gnu/32bit; dynamically linked.
# >Number:         35
# >Category:       Monet_SGI
# >Synopsis:       BATloop semantics
# >Confidential:   no
# >Severity:       serious
# >Priority:       high
# >Responsible:    mk
# >State:          feedback
# >Class:          sw-bug
# >Submitter-Id:   unknown
# >Arrival-Date:   Tue Sep 30 20:45:03 1997
# >Originator:     mk
# >Organization:
# >Release:        970101
# >Environment:
# SGI
# >Description:
# Since yesterday the semantics has changed?
# > b:= view_bbp_name;
# > b@batloop(){ printf("%s\n",$t);}
# !ERROR: interpret: no matching iterator to 'batloop(bat[int,str], ptr)'.

var b:= view_bbp_name();
b@batloop(){ printf("%s\n",$t);}
monet_environment
monet_fcn_fcn
monet_fcn_imp
monet_fcn_tpe
monet_fcn_mid
monet_fcn_sig
monet_fcn_dsc
monet_fcn_aut
monet_fcn_dat
monet_fcn_nme
monet_fcn_pro
monet_fcn_cnt
monet_mod_nme
monet_mod_cnt
monet_mod_atm
monet_mod_acc
monet_mod_hdl
monet_mod_dep
monet_mod_use
monet_mod_load
monet_mod_drop
monet_mod_proc
monet_mod_var
adm_fcntbl
adm_acctbl
adm_atomtbl
monet_unicode_case

# 13:14:58 >  
# 13:14:58 >  Done.
# 13:14:58 >  

